index.js

```diff
@@ -0,0 +1,94 @@
+const BASE_URL = 'https://trektravel.herokuapp.com';
+$(document).ready(() => {
+
+  $('#load').click(function(response) {
+
+    let fullUrl = BASE_URL + '/trips'
+    $.get(fullUrl, response => {
+      console.log(response);
+
+      response.forEach((trip) => {
+        let name = trip.name,
+        id = trip.id,
+        continent = trip.continent,
+        weeks = trip.weeks;
+        let allTripInfo = `<li data-id=${id}><h3>${name}:</h3> <p> Location: ${continent} Length: ${weeks} weeks. </p></li>`;
+        $('#all-trips ul').append(allTripInfo);
+      });
+
+      $('#all-trips h3').on('click', onTripClick)
+    })
+    .fail(function(response){
+      console.log(response);
+      console.log('failure');
+      $('#message').html(`<p>Error!</p>`)
+    })
+    .always(function(){
+      console.log('always even if we have success or failure');
+    });
+  });
+
+  // begin .get to see more info on individual trips
+  const onTripClick = function onTripClick() {
+    const tripLi = $(this.parentElement)
+    let tripID = tripLi.attr('data-id');
+    let individualURL = `${BASE_URL}/trips/${tripID}`
+
+    $(this).click((event) => {
+      event.stopPropagation();
+    }); // stopping click from running too many times
+
+    $.get(individualURL, response => {
+
+      tripLi.append(`
+        <p>ID: ${response.id}</p>
+        <p>Category: ${response.category}</p>
+        <p>Destination: ${response.continent} </p>
+        <p>About: ${response.about} </p>
+        <p>Cost: $${response.cost}</p>
+        <p>~</p>`);
+        tripLi.append('<p class="button"> Reserve Today! </p>');
+
+        $(this).one('click', 'p', function() {
+          let formInfo = `<form id="add-reservation" action="${individualURL}/reservations">
```

Comment:
This did not work for me - clicking the button did not display the reservation form.

Your code here looks mostly OK, except that you're trying to register a click handler on `$(this)`. Right now you're inside a success callback for an AJAX request (the `$.get` on line 41), but `$(this)` is only set to a DOM element if you're in a jQuery event handler. I believe you would get the effect you want if you were to structure the above code as:

```javascript
// Make a jQuery DOM element. It's not yet attached to the page, but we can
// still listen for events on it.
const reserveButton = $('<p class="button">Reserve Today!</p>');

// Attach the element to the DOM
tripLi.append(reserveButton);

// Listen for the click event
reserveButton.on('click', function() {
```